Common/Wrapper/TraktManager.swift

```diff
@@ -107,8 +116,21 @@ public class TraktManager {
         self.session = session
     }
     
-    // MARK: - Setup
+    // MARK: - Setup Clients
+    public func setOauth2RedirectURL(withClientID: String, clientSecret secret: String, redirectURI: String, staging: Bool = false) {
+        self.clientID = withClientID
+        self.clientSecret = secret
+        self.redirectURI = redirectURI
+        
+        self.staging = staging
+        
+        self.baseURL = !staging ? "trakt.tv" : "staging.trakt.tv"
+        self.APIBaseURL = !staging ? "api.trakt.tv" : "api-staging.trakt.tv"
+        self.oauthURL = URL(string: "https://\(baseURL!)/oauth/authorize?response_type=code&client_id=\(withClientID)&redirect_uri=\(redirectURI)")
+    }
```

Comment:
I think this should be reverted. `set` could use a better signature but I'm iffy on `setOauth2RedirectURL`, this can be a different PR / change to the tvOS support.

Reply:
Sure, no problem. I just wanted to give it a different name because "set" was a bit too generic.